aldryn_newsblog/utils/utilities.py

```diff
@@ -148,3 +151,38 @@ def add_prefix_to_path(path, prefix):
         return "{0}/{1}".format(prefix, splitted_path[0])
     # directory/template.html => directory/prefix/template.html
     return "{0}/{1}/{2}".format(splitted_path[0], prefix, splitted_path[1])
+
+
+def is_valid_namespace(namespace):
+    """
+    Check if provided namespace has an app-hooked page.
+    Returns True or False.
+    """
+    try:
+        reverse('{0}:article-list'.format(namespace))
```

Comment:
curious.. how can this throw an ``AttributeError``? If the namespace is empty and django somewhere in reverse expects it to have something?